elastic/integrations
# [Oracle] Mark `password` field as secret

## Proposed commit message
The PR contains changes to mark the password field as secret. 

## Checklist


## Author's Checklist


- [x] Mark the password field as secret
- [x] Upgrading the package doesn't break the existing integration and metrics still come through
- [x] No dashboards are broken

## How to test this PR locally
- Spin up the Elastic stack and point your `elastic-package` to the running Kibana instance
- Build the package locally and install the package using `elastic-package install --zip oracle-1.25.1.zip`
- Upgrade the integration to the latest version i.e. `1.25.1`, there shouldn't be any errors. If you find any errors while upgrading, please report it by adding a comment to this PR.
- You should now see separate fields for `username` and `password` under advanced options while configuring the Integration for metrics collection
- Take a look at the dashboards, they shouldn't be broken and metrics should still be coming in just fine


## Related issues
- Relates to https://github.com/elastic/obs-infraobs-team/issues/1369

## Screenshots
These are after enabling the `Performance` and `Memory` metricsets. 

![image](https://github.com/elastic/obs-infraobs-team/assets/38116245/63eba9ef-08af-42df-908d-c76e4b68163e)
*Oracle Metrics Overview Dashboard*

![image](https://github.com/elastic/obs-infraobs-team/assets/38116245/b595fc86-71dc-49e4-ab5e-f2dcef154c05)
*Performance Metrics Dashboard*

![image](https://github.com/elastic/obs-infraobs-team/assets/38116245/84309846-82fc-422b-8ee1-89998a697d81)
*Memory Metrics Dashboard*
